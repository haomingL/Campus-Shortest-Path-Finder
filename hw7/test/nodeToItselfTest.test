# This test tests the appropriate behavior of path from one node to itself
LoadGraph people people.tsv

FindPath people Haoming-Liu Haoming-Liu