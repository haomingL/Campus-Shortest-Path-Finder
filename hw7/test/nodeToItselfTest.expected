# This test tests the appropriate behavior of path from one node to itself
loaded graph people

path from Haoming-Liu to Haoming-Liu:
total cost: 0.000